libsovtoken/src/logic/api_internals/add_request_fees.rs

```diff
@@ -1,6 +1,6 @@
 //! TODO ???
 
```

Comment:
I would like to see this source file commented to keep consistent with other files and maintain the generated documentation

Reply:
I'm afraid I am not the best person to describe sovtoken files.